packages/components/src/ui/styles/test/styled.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import { css, styled } from '@wp-g2/styles';
+import { render } from '@testing-library/react';
+
+describe( 'basic', () => {
+	test( 'should create a styled component', () => {
+		const Component = styled.div`
+			background: blue;
+		`;
+
+		const { container } = render( <Component /> );
+
+		expect( container.firstChild ).toHaveStyle( { background: 'blue' } );
+	} );
+
+	test( 'should style an existing component', () => {
+		const Previous = ( props ) => <div { ...props } />;
+
+		const Component = styled( Previous )`
+			background: blue;
+		`;
+
+		const { container } = render( <Component /> );
+
+		expect( container.firstChild ).toHaveStyle( { background: 'blue' } );
+	} );
+
+	test( 'should style an existing styled component', () => {
+		const Previous = styled.div`
+			color: red;
+		`;
+
+		const Component = styled( Previous )`
+			background: blue;
+		`;
```

Comment:
Is it worth testing the same CSS property for both of these in a separate test to prove that the order of application is correct?

Reply:
Oooo! Good idea :D